packages/storybook/stories/GridLayout/GridLayout.stories.tsx

```diff
@@ -79,16 +79,30 @@ export const GridLayout: () => JSX.Element = () => {
 
   const participants = object('Participants', defaultParticipants);
 
-  const participantsComponents = participants.map((participant, index) => {
-    return (
-      <VideoTile
-        isVideoReady={participant.isVideoReady}
-        renderElement={<StreamMedia videoStreamElement={participant.isVideoReady ? renderVideoStream() : null} />}
-        displayName={participant.displayName}
-        key={index}
-      />
-    );
-  });
+  const videoStreamElements = useVideoStreams(
+    participants.filter((participant) => {
+      return participant.isVideoReady;
+    }).length
+  );
+
+  const participantsComponents = useMemo(() => {
```

Comment:
nit `useCallback`?

Reply:
This returns an object instead of a function so I believe we should use useMemo here but please let me know if I'm mistaken.